docker-compose.yaml

```diff
@@ -0,0 +1,28 @@
+name: gitalytics
+
+services:
+#  frontend:
+#    build: ./src/frontend
+#
+#  fastapi-server:
+#    build: ./src/backend
+#    ports:
+#      - "8000:8000" # change internal port to 60860 (random port))
+#    depends_on:
+#      database:
+#        condition: service_healthy
+#      frontend:
+#        condition: service_completed_successfully
```

Comment:
auskommentiert, weil es noch nicht klappt?

Reply:
Genau, da müsste man noch einiges mit docker networking korregieren. Zudem habe ich es nicht hinbekommen, dass die container in der richtigen Reihenfolge starten...